http-server-upload.js

```diff
@@ -208,6 +208,7 @@ server.on('request', (req, res) => {
 
     });
   } else {
+    let formId="fileinput";
```

Comment:
Why this variable and not just a fixed ID in the HTML part?

Reply:
I did that to highlight it isn't just a throwaway id, but yeah, that was too obvious.